baseapp/baseapp_test.go

```diff
@@ -221,11 +221,13 @@ const msgType = "testUpdatePowerTx"
 func (tx testUpdatePowerTx) Type() string                            { return msgType }
 func (tx testUpdatePowerTx) Get(key interface{}) (value interface{}) { return nil }
 func (tx testUpdatePowerTx) GetMsg() sdk.Msg                         { return tx }
-func (tx testUpdatePowerTx) GetSignBytes() []byte                    { return nil }
+func (tx testUpdatePowerTx) GetSignBytes(sdk.Context) []byte         { return nil }
 func (tx testUpdatePowerTx) ValidateBasic() sdk.Error                { return nil }
 func (tx testUpdatePowerTx) GetSigners() []crypto.Address            { return nil }
 func (tx testUpdatePowerTx) GetFeePayer() crypto.Address             { return nil }
 func (tx testUpdatePowerTx) GetSignatures() []sdk.StdSignature       { return nil }
+func (tx testUpdatePowerTx) GetAccNonce() int64                      { return 0 }
+func (tx testUpdatePowerTx) GetTxNonce() int64                       { return 0 }
```

Comment:
Let's keep the nonces in StdSignature, then we don't have to add these methods

Reply:
Then you have one StdSignature with nonces, and the rest without.  Nonces are a piece of data that has to do with the tx, not the signature.